reactrestaurant/src/components/accordian.js

```diff
@@ -0,0 +1,58 @@
+/*
+
+*/
+
+import React from 'react';
+
+function Accordian(props){
+ 
+    return(
+<div className="accordion" id="accordionExample">
```

Comment:
This is a good opportunity to leverage what React is great at. In following good DRY (Don't repeat yourself) principles, let's consider how this could be broken up into a repeatable component where you can pass the relevant information into it.